File: server/herehear/src/main/java/com/ssafy/herehear/db/entity/Library.java
package com.ssafy.herehear.db.entity;

import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class Library extends BaseEntity {
	@ManyToOne(fetch = FetchType.LAZY)
	@JoinColumn(name = "user_id")
	private Account account;
	
	@ManyToOne(fetch = FetchType.LAZY)
	@JoinColumn(name = "book_id")
	private Book book;
	
	private int read_status;
	private int stars;
	private Boolean flag;
}

File: client/herehear/app/src/main/java/com/ssafy/herehear/model/local/entity/Library.kt
package com.ssafy.herehear.model.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Library")
data class Library(
    @PrimaryKey
    val id: String,
    val book_id: Int,
    val img_url: String,
    val read_status: Int,
    val stars: Int
)
